### PR TITLE
Removing (some) common globals

### PR DESCRIPTION
**Pull request by gebart**
Started working on fixing #2346

### CODE REVIEW

**Comment by LudwigOrtmann** [comment]
Can you specify the intended scope of this PR?

**Comment by gebart** [comment]
I decided to begin fixing the places where there were more than one definition of the same variable name, but I had no real plan. I'm currently fiddling with this while waiting in a phone queue... Do you want to fix some of these?

**Comment by LudwigOrtmann** [comment]
I feel responsible for native ;)

**Comment by gebart** [comment]
We might as well merge this in steps to avoid huge changes. Travis is happy, who is suitable to ACK this PR?

**Comment by LudwigOrtmann** [comment]
@authmillenon you might want to have a peek as well.

**Comment by LudwigOrtmann** [comment]
Merging in smaller chunks seems reasonable to me.

File: sys/net/routing/etx_beaconing.c [hunk 1]
```diff
@@ -103,12 +103,12 @@ static etx_neighbor_t candidates[ETX_MAX_CANDIDATE_NEIGHBORS];
...
-//Transceiver command for sending ETX probes
-transceiver_command_t tcmd;
+static mutex_t etx_mutex = MUTEX_INIT;
+/* Transceiver command for sending ETX probes */
+static transceiver_command_t tcmd;
 
-//Message to send probes with
-msg_t mesg;
+/* Message to send probes with */
+static msg_t mesg;
```
Comment (LudwigOrtmann) [inline sys/net/routing/etx_beaconing.c]: Is this used at all?

File: sys/net/routing/etx_beaconing.c [hunk 2]
```diff
@@ -103,12 +103,12 @@ static etx_neighbor_t candidates[ETX_MAX_CANDIDATE_NEIGHBORS];
  * In this time, no packet may be handled, otherwise it could assume values
  * from the last round to count for this round.
  */
-mutex_t etx_mutex = MUTEX_INIT;
-//Transceiver command for sending ETX probes
-transceiver_command_t tcmd;
+static mutex_t etx_mutex = MUTEX_INIT;
+/* Transceiver command for sending ETX probes */
+static transceiver_command_t tcmd;
```
Comment (LudwigOrtmann) [inline sys/net/routing/etx_beaconing.c]: Is this used at all?

**Comment by LudwigOrtmann** [comment]
ACK except for the comments above.

File: sys/net/routing/etx_beaconing.c (hunk 1 above)
Comment (gebart) [inline sys/net/routing/etx_beaconing.c]: Doesn't seem like it..

File: sys/net/routing/etx_beaconing.c (hunk 2 above)
Comment (gebart) [inline sys/net/routing/etx_beaconing.c]: Doesn't seem like it..

**Comment by gebart** [comment]
Deleted two unused variables in etx_beaconing.c, waiting for Travis.

**Comment by gebart** [comment]
Travis is happy, comments addressed, merging.

**Pull request merged**